### PR TITLE
1st draft wee_import instructions

### PR DESCRIPTION
**Pull request by gjr80**
Raised this PR for some discussion on structure of the `wee_import` instructions. `wee_import` is  somewhat more complex than some of the other utilities and hence the new `wee_import` section is somewhat longish. I vasilated for some time over how to structure it; in the end I went with the following outline:

- Importing data

    * why use it
    * --help printout
    * command line options (valid settings and deafults where applicable)

- CSV data

    * pre-requisites/file structure
    * step-by-step to import CSV data
    * required `wee_import.conf` settings
    * command line example
    * exected output
    * what to do if it goes wrong

- WU data

    * what fields will WU populate
    * step-by-step to import WU data
    * required `wee_import.conf` settings
    * command line example
    * exected output
    * what to do if it goes wrong

- Cumulus data

    * what fields will Cumulus populate
    * step-by-step to import Cumulus data
    * required `wee_import.conf` settings
    * command line example
    * exected output
    * what to do if it goes wrong
    
- `wee_import.conf` settings explained

I appreciate it may be a bit all over the place but I tried to structure it so the user would get fairly quickly to what they need (ie the step-by-step instructions) without the need to go through dry config setting/comand line option explanations (which we know they won't read).

Would appreciate any thoughts on the structure. In terms of content it is first draft and I need to go through it again a few more times tidying up the content.

Oh, I have made a few minor changes to some config options which I will commit later, so if you get down to that level of detail and some things don't agree they will be fixed in due course.

### CODE REVIEW

File: docs/customizing.htm [hunk 1]
```diff
@@ -5149,6 +5149,1577 @@ <h2 id="Changing_the_unit_system">Changing the unit system</h2>
         or they can be rebuilt with the tool:</p>
     <pre class="tty cmd">wee_database weewx.conf --backfill-daily</pre>
 
+    <h1 id="import">Importing observation data</h1>
+
+    <p>Some <span class="code">weewx</span> users will have historical data from another source (eg other weather station software or a manually compiled file) that they wish to import into the <span class="code">weewx</span> database. Such data can, depending upon the source, be imported automatically using the <span class="code">wee_import</span> utility. This section details the use of the <span class="code">wee_import</span> utility.</p>
```
Comment (tkeffer) [inline docs/customizing.htm]: The word 'that' in this context is acting as a restrictive pronoun (Says Struck & White, "The lawnmower *that* is in the shed."). You want "which"

File: docs/customizing.htm [hunk 2]
```diff
@@ -5149,6 +5149,1577 @@ <h2 id="Changing_the_unit_system">Changing the unit system</h2>
...
+        below:</p>
+
+    <h4><span class="config_option">--config</span></h4>
+    
+    <p>The utility is pretty good at "guessing" where your configuration file 
+        <span class="code">weewx.conf</span> is, but if you've done an unusual 
+        install, you may have to tell it explicitly. You can do this by using 
+        the <span class="config_option">--config</span> option:</p>
+    
+    <pre class="tty">wee_import -csv --config=/this/folder/weewx.conf --source=/var/tmp/data.csv
```
Comment (tkeffer) [inline docs/customizing.htm]: s/-csv/--csv/

File: docs/customizing.htm [hunk 3]
```diff
@@ -5149,6 +5149,1577 @@ <h2 id="Changing_the_unit_system">Changing the unit system</h2>
...
+    <p>or</p>
+    
+    <pre class="tty">wee_import -cumulus --source=/var/tmp/cumulus</pre>
+                        
+    <h4><span class="config_option">--calc-missing</span></h4>
+    
+    <p><span class="code">wee_import</span> can attempt to calculate any missing 
+        derived observations, (eg dew point, rain rate, apparent temperature etc) 
+        from the imported data. If the 
+        <span class="config_option">--calc-missing</span> option is used
```
Comment (tkeffer) [inline docs/customizing.htm]: Comma after "is used"

File: docs/customizing.htm [hunk 4]
```diff
@@ -5149,6 +5149,1577 @@ <h2 id="Changing_the_unit_system">Changing the unit system</h2>
...
+        imported. It is advisable to use the full path when specifying the file 
+        or folder name. If specifying a folder name do not include a trailing 
+        <span class="code">/</span>. This option is mandatory when importing 
+        data from one or more files.</p>
+    
+    <pre class="tty">wee_import -csv --source=/var/tmp/data.csv</pre>
+
+    <p>or</p>
+    
+    <pre class="tty">wee_import -cumulus --source=/var/tmp/cumulus</pre>
```
Comment (tkeffer) [inline docs/customizing.htm]: s/-cumulus/--cumulus/g

**Comment by tkeffer** [comment]
Wow! You put in a lot of work here!

I found it pretty easy to follow. 

I imagine a lot of this would change if you follow Matthew's suggestion of a different import file for each type of import.

**Comment by gjr80** [comment]
>Wow! You put in a lot of work here!
>
>I found it pretty easy to follow. 

The above was all that popped upon on my iPad, and I thought 'yes, but given this mornings #97 conversation that might change'

Ok, at least the structure seems to flow which was the main comment I was after. Though I wonder if Matthew has any views..... :)

**Comment by gjr80** [comment]
Since `wee_import` has undergone a bit of a restructure I am going to close this PR, create a `wee_import` branch in `weewx:weewx` and will resubmit later (when its rewritten).

**Pull request closed without merging**